ansible/roles/biocache3-properties/templates/biocache-config.properties

```diff
@@ -464,6 +464,10 @@ security.jwt.enabled={{ jwt_auth_enabled | default('false') }}
 security.jwt.discovery-uri={{ oidc_discovery_url }}
 security.jwt.clientId={{ (biocache_client_id | default(oidc_client_id)) | default('') }}
 
+webservice.client-id = {{ (biocache_client_id | default(oidc_client_id)) | default('') }}
+webservice.client-secret = {{ (biocache_client_secret | default(oidc_client_secret)) | default('') }}
+webservice.jwt-scopes = {{ (biocache_jwt_scopes | default(oidc_jwt_scopes)) | default('openid,email') }}
```

Comment:
Remove or refactor. See current `master` branch.

Reply:
I've just resolved the conflicts with master.